src/components/UserInfo.js

```diff
@@ -10,9 +11,18 @@ export default class UserInfo {
       job: this._jobElement.textContent,
     };
   }
+  getUserId() {
+    return this._userId;
+  }
 
-  setUserInfo({ name, job }) {
+  setUserInfo({ name, job, avatar, id }) {
     this._nameElement.textContent = name;
     this._jobElement.textContent = job;
+    this._avatarElement.src = avatar;
```

Comment:
`   this._avatarElement.src = avatar;`


Please, reuse the `setUserAvatar` method here, to avoid code duplication


```js

    this.setUserAvatar(avatar)

```